R/derive_merged.R

```diff
@@ -160,6 +160,69 @@
 #'   match_flag = vsdatafl
 #' ) %>%
 #'   select(STUDYID, USUBJID, AGE, AGEU, LASTWGT, LASTWGTU, vsdatafl)
+#'
+#' # derive treatment start datetime (TRTSDTM)
```

Comment:
Comments should start with proper case.  
```suggestion
#' # Derive treatment start datetime (TRTSDTM)
```

Reply:
Updated